src/resources/coupons-resource.js

```diff
@@ -1,53 +1,52 @@
-export default function CouponsResource({apiHandler}) {
-    return {
-        getAll({limit = null, offset = null, sort = null, filter = null, q = null} = {}) {
-            const params = {
-                limit,
-                offset,
-                sort,
-                filter,
-                q
-            };
-            return apiHandler.getAll(`coupons`, params);
-        },
-
-        get({id}) {
-            return apiHandler.get(`coupons/${id}`);
-        },
-
-        create({id = '', data}) {
-            return apiHandler.create(`coupons/${id}`, id, data);
-        },
-
-        update({id, data}) {
-            return apiHandler.put(`coupons/${id}`, data);
-        },
-
-        getAllRedemptions({limit = null, offset = null, sort = null, filter = null, q = null} = {}) {
-            const params = {
-                limit,
-                offset,
-                sort,
-                filter,
-                q
-            };
-            return apiHandler.getAll(`coupons-redemptions`, params);
-        },
+/**
+*This file was auto-generated by our auto-sdk-code-generator.
+* Do not make direct changes to this file.
+*/
 
-        getRedemption({id}) {
-            return apiHandler.get(`coupons-redemptions/${id}`);
-        },
-
-        cancelRedemption({id}) {
-            return apiHandler.post(`coupons-redemptions/${id}/cancel`);
-        },
-
-        redeem({data}) {
-            return apiHandler.post(`coupons-redemptions`, data);
-        },
-
-        setExpiration({id, data}) {
-            return apiHandler.post(`coupons/${id}/expiration`, data);
-        }
-    };
-};
+export default function CouponsResource({apiHandler}) {
+  return {
+    getAllRedemptions({
+      expand = null,
+      limit = null,
+      offset = null,
+      filter = null,
+      q = null,
+      sort = null,
+    } = {}) {
+      const params = {limit, offset, filter, q, sort, expand};
```

Comment:
I'm curios is there any concrete reason for this constant? I mean could we just construct object in next line?